src/main/resources/db/migration/claimstore/V2018_07_16_1318__Update_response_payment_option_for_full_by_set_date.sql

```diff
@@ -0,0 +1,5 @@
+UPDATE
+  claim
+SET
+  response = JSONB_SET(response, '{paymentOption}',  '"BY_SPECIFIED_DATE"' :: JSONB)
+WHERE response IS NOT NULL AND jsonb_extract_path_text(response, 'paymentOption') = 'FULL_BY_SPECIFIED_DATE';
```

Comment:
does ccj need to be updated too?

Reply:
Yes, it looks like the same enum is used in CCJ's